tests/unit_tests/test_flight_boundaries_container.py

```diff
@@ -0,0 +1,12 @@
+from SDA import *
+import numpy as np
+from SUASSystem import *
+import unittest
+
+class FlightBoundaryContainerTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.boundary = FlightBoundariesContainer(np.array([np.array([[-2000, -2000], [-2000, 2000], [2000, 2000], [2000, -2000]])]))
+
+    def test_is_point_in_bounds(self):
+        self.assertTrue(self.boundary.is_point_in_bounds(np.array([0,0,0])))
```

Comment:
You need additional tests here. This is a very important method in the SDA process, and we use a complex path at competition. While it is great to test a simple path like ```[[-2000, -2000], [-2000, 2000], [2000, 2000], [2000, -2000]]```, you need to test more complex paths as well (non regular polygons).

Additionally, you need multiple tests for each flight boundary. I'd recommend picking 3-4 points in each boundary path and testing those.

You also need to test multi-boundary flight containers (same class, different parameter length). At competition, we have multiple flight boundaries which are all put into this one class. This class needs to be able to determine if a point exists in any of the boundaries.

Reply:
done